Tests/BraveWalletTests/SwapTokenStoreTests.swift

```diff
@@ -64,36 +64,6 @@ class SwapStoreTests: XCTestCase {
     }
   }
 
-  func testDefaultSellBuyTokensOnRopstenWithoutPrefilledToken() {
-    let rpcService = MockJsonRpcService()
-    let store = SwapTokenStore(
-      keyringService: MockKeyringService(),
-      blockchainRegistry: MockBlockchainRegistry(),
-      rpcService: rpcService,
-      swapService: MockSwapService(),
-      txService: MockTxService(),
-      walletService: MockBraveWalletService(),
-      ethTxManagerProxy: MockEthTxManagerProxy(),
-      prefilledToken: nil
-    )
-    let ex = expectation(description: "default-sell-buy-token-on-ropsten")
-    XCTAssertNil(store.selectedFromToken)
-    XCTAssertNil(store.selectedToToken)
-
-    rpcService.setNetwork(BraveWallet.RopstenChainId, coin: .eth) { success in
-      XCTAssertTrue(success)
-      let testAccountInfo: BraveWallet.AccountInfo = .init()
-      store.prepare(with: testAccountInfo) {
-        defer { ex.fulfill() }
-        XCTAssertEqual(store.selectedFromToken?.symbol.lowercased(), "eth")
-        XCTAssertEqual(store.selectedToToken?.symbol.lowercased(), "dai")
-      }
-    }
-    waitForExpectations(timeout: 3) { error in
-      XCTAssertNil(error)
-    }
-  }
-
```

Comment:
removed unit test since no longer a case

Reply:
> Can we not keep this test using `mockGoerli` / `mockSepolia`?

There wont be prefilled token for goerli network. and swap is not available for sepolia network.